.buildkite/core.rayci.yml

```diff
@@ -1,179 +1,179 @@
 group: core tests
 steps:
-  # build
-  - name: corebuild
-    wanda: ci/docker/core.build.wanda.yaml
-    depends_on: oss-ci-base_build
+  # # build
```

Comment:
what are these?

you can just add tag "disabled" to skip.

Reply:
oh, i'm still testing the change and don't want to trigger all tests